docs/docs/seccomp.md

```diff
@@ -0,0 +1,58 @@
+---
+title: seccomp
+---
+
+Secure computing mode (seccomp) is a Linux kernel feature used to restrict the actions available within a running container. The seccomp() system call operates on the seccomp state of the calling process. You can use this feature to restrict your application’s access to the underlying system.
```

Comment:
I might just be dumb but I have no idea what the middle sentence is trying to tell me